src/pages/WorkloadDetails/WorkloadPodLogs.tsx

```diff
@@ -731,27 +701,28 @@ class WorkloadPodLogs extends React.Component<WorkloadPodLogsProps, WorkloadPodL
   };
 
   private getContainers = (pod: Pod): Container[] => {
-    // consistently position the proxy container first, if it (they?) exist.
+    // sort containers by name, consistently positioning proxy container first.
     let podContainers = pod.istioContainers || [];
+    podContainers.push(...(pod.containers || []));
+    podContainers = podContainers.sort((c1, c2) => {
+      if (c1.isProxy !== c2.isProxy) {
+        return c1.isProxy ? 0 : 1;
+      }
+      return c1.name < c2.name ? 0 : 1;
+    });
+    let appContainers = 0;
     let containers = podContainers.map(c => {
       const name = c.name;
-      const displayName = c.name;
+      if (c.isProxy) {
+        return { color: proxyContainerColor, displayName: name, isProxy: true, isSelected: true, name: name };
+      }
+      const version = pod.appLabel && pod.labels ? pod.labels[serverConfig.istioLabels.versionLabelName] : undefined;
+      const displayName = !version ? name : `${name}-${version}`;
```

Comment:
I see this confusing:
![image](https://user-images.githubusercontent.com/1662329/118251043-64e1f480-b4a7-11eb-89df-8c8baa584e5b.png)

You're appending a version label to the container name and that's not really true.
I'd keep with the container name because it's what I expect to see as a user:

```
      containers:
        - name: cars
          image: quay.io/kiali/demo_travels_cars:v1
```

Appending the version at the container level is not helping here.
(Note that pod is always visible)

Reply:
I'm not sure, I think that is a holdover from long ago and not new to this PR.  But I agree with you, I'm not sure why it's like that and I'll change it.